### DISCUSSION TITLE
Changing the Lua path for libraries?

### DISCUSSION THREAD

**Discussion by lmartinez-mirror** [sammy-ette/Hilbish#79] [Q&A] [answered]
Hi, I'm in the middle of refactoring Hilbish's dependencies on the AUR. Is there a way to change the paths Hilbish searches for the libraries at compile time (e.g. `/usr/share/lua/5.4/` instead of `/usr/share/hilbish/libs/`)? If so, how?

**Comment by sammy-ette** [comment, marked as answer]
hilbish already looks for lua libraries in the standard path like the one you described (in addition to the custom ones),
except its for 5.1 since the lua interpreter for hilbish is only 5.1
if you still want to change it, change either `dataDir` or `requirePaths` via the -X flag of go build

**Reply by lmartinez-mirror** [reply]
Thank you for your quick response! I'll keep this in mind while refactoring the packages.
